Google.Api.Gax.Grpc/CallSettingsExtensions.cs

```diff
@@ -70,21 +70,20 @@ public static CallSettings MergedWith(this CallSettings original, CallSettings o
         /// merged with the (optional) original settings specified by <paramref name="settings"/>.
         /// </summary>
         /// <param name="settings">Original settings. May be null, in which case the returned settings
-        /// will only contain call timing.</param>
-        /// <param name="expiration">Expiration to use in the returned settings, possibly as part of a retry. Must not be null.</param>
-        /// <returns>A new set of call settings, or null if both parameters are null.</returns>
+        /// will only contain the expiratoin.</param>
+        /// <param name="expiration">Expiration to use in the returned settings, possibly as part of a retry. May be null,
+        /// in which case any expiration in <paramref name="settings"/> is not present in the new call settings. If
```

Comment:
I'm not sure this is the most suitable semantics for `expiration` being `null`. We have `Expiration.None` to mean no expiration; to my mind, null should mean it passes through the expiration from `settings`?